crates/e2e/src/setup/colocation.rs

```diff
@@ -102,11 +102,31 @@ graph-url = "{subgraph}"
     }
 }
 
+pub enum LiquiditySourceNotifier {
+    Liquorice { api_port: u16 },
+}
+
+impl LiquiditySourceNotifier {
+    pub fn to_config_string(&self) -> String {
+        match self {
+            Self::Liquorice { api_port } => format!(
+                r#"
+[liquidity-sources-notifier.liquorice]
+base-url = "http://0.0.0.0:{api_port}"
+api-key = "none"
+http-timeout = "10s"
+"#,
+            ),
+        }
+    }
+}
+
 pub fn start_driver(
     contracts: &Contracts,
     solvers: Vec<SolverEngine>,
     liquidity: LiquidityProvider,
     quote_using_limit_orders: bool,
+    liquidity_source_notifier: Option<LiquiditySourceNotifier>,
```

Comment:
@MartinquaXD I had to add this configuration for the new `e2e` test, but it affected a bunch of other files.

We may add an additional function (`start_driver_with_config` or smth like that), and take care of preliminary refactorings. Please let me know if it's needed.